content/installation/migrating-manager.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: bt_wiki
+title: Reinstalling and Migrating Cloudify Manager
+category: Installation
+draft: true
+weight: 400
+
+---
+
+To move your Cloudify Manager to another host, you must use same process as a reinstall. One reason to migrate is to move your Cloudify Manager to a host with more resources. You can do this in one of these ways:
+
+* **In-place reinstall** - Replace Cloudify Manager with the same version on the same host
+* **Migration reinstall** - Replace Cloudify Manager with the same version on another host
+
+The reinstall process includes:
+
+* In-place reinstall -
+
+  1. Backup a snapshot of the Cloudify Manager.
+  1. Backup the agents certificates from the Cloudify Manager.
+  1. Uninstall the Cloudify Manager from the host
+  1. Install the Cloudify Manager on the host.
+  1. Restore the snapshot of the Cloudify Manager to the host.
+
+* Migration reinstall -
+
+  1. Backup a snapshot of the Cloudify Manager.
+  1. Backup the agents certificates from the Cloudify Manager.
+  1. Install the Cloudify Manager on the host.
+  1. Restore the snapshot of the Cloudify Manager to the host.
+  1. (Optional) Uninstall the Cloudify Manager from the original host. In Cloudify 4.2 and below, this is called teardown.
+  1. Migrate agents from the old Cloudify Manager.
```

Comment:
I would recommend after this stage that the old manager be shut down, even if it is not to be deleted. If any auto-heal policies are in use then the old manager MUST also be disconnected from any networks on which it could reach the platforms it is managing. Otherwise, after the first auto-heal occurs, you will end up with the old manager auto-healing as well, and thus get two copies of compute nodes for a deployment, each orphaned from one of your managers.

Reply:
I'll add a warning: If you do not uninstall the old Cloudify Manager and your workflows include auto-heal workflows, you must disconnect it from any networks that it can use to communicate with managed hosts.